lib/msf/core/post/file.rb

```diff
@@ -153,6 +153,19 @@ def setuid?(path)
     end
   end
 
+  #
+  # See if +path+ on the remote system exists and is executable
+  #
+  # @param path [String] Remote path to check
+  #
+  # @return [Boolean] true if +path+ exists and is executable
+  #
+  def executable?(path)
```

Comment:
Very nice. I was hoping you'd add this method and use `readable?` when I saw #12611. I don't have to request anything now. :)